src/components/Recipes/recipesWithSearch/index.jsx

```diff
@@ -2,32 +2,48 @@ import { useEffect, useState } from "react";
 import { API_URL } from "../../../constants";
 import { Search, Loading, RecipesFeed, SelectRecipesType } from "../..";
 
-export const RecipesWithSearch = ({ urlParams, useRecipe, mode, userId = null }) => {
+export const RecipesWithSearch = ({
+  urlParams,
+  useRecipe,
+  mode,
+  userId = null,
+}) => {
   const [currentPage, setCurrentPage] = useState(1);
   const [search, setSearch] = useState([]);
-  const [category, setCategory] = useState('all-recipes');
+  const [category, setCategory] = useState("all-recipes");
   const { data, isLoading } = useRecipe(category, currentPage, userId);
-  const userIdSuffix = (urlParams === 'savedRecipes'||urlParams==='user') ? `/${userId}` : null;
+  const userIdSuffix =
+    urlParams === "savedRecipes" || urlParams === "user" ? `/${userId}` : null;
   const dynamicUrl = `${API_URL.RECIPES.SEARCH.RECIPES}/${urlParams}/${category}/${currentPage}${userIdSuffix}`;
-  const totalPages = Math.ceil(search?.totalRecipesCount > 0 ? search?.totalRecipesCount / 9 : data?.totalRecipesCount / 9);
+  const totalPages = Math.ceil(
+    search?.totalRecipesCount > 0
+      ? search?.totalRecipesCount / 9
+      : data?.totalRecipesCount / 9
+  );
```

Comment:
The number 9 should probably be inside a constants file, I assume this represents the number of pages? so for example you can call it `RECIPES_PER_PAGE` or anything else